assets/queries/ansible/azure/azure_instance_using_basic_authentication/query.rego

```diff
@@ -0,0 +1,24 @@
+package Cx
+
+import future.keywords.if
+
+CxPolicy[result] {
+	vm := input.document[i].playbooks[k].azure_rm_virtualmachine
+    is_linux_vm(vm)
+    not vm.ssh_password_enabled == false
```

Comment:
```suggestion
    not vm.linux_config.disable_password_authentication == false
```

Reply:
@cxMiguelSilva I can't seem to find `linux_config` and `disable_password_authentication` in the Ansible documentation, are you positive this is correct configuration?
https://docs.ansible.com/ansible/2.7/modules/azure_rm_virtualmachine_module.html

Reply:
I have found it here
https://docs.ansible.com/ansible/latest/collections/azure/azcollection/azure_rm_virtualmachine_module.html#parameter-linux_config/disable_password_authentication
Let me know what you think

Reply:
I see that's a newer version of Ansible.. I'll edit the rule to include both cases then

Reply:
Ok thank you so much!